src/main/java/com/klarna/hiverunner/builder/HiveShellBase.java

```diff
@@ -100,7 +102,21 @@
     }
     
     private List<Object[]> executeStatementWithCommandShellEmulation(String hql) {
-      return hiveServerContainer.executeStatement(commandShellEmulation.transformStatement(hql));
+      String transformedHql = commandShellEmulation.transformStatement(hql);
+      if (commandShellEmulation.isImportFileStatement(transformedHql)) {
+        return importScript(transformedHql);
+      }
+      return hiveServerContainer.executeStatement(transformedHql);
+    }
+
+    private List<Object[]> importScript(String hql) {
+      String[] tokens = hql.trim().split("\\s+");
+      String fileName = null;
+      if (tokens.length == 2) {
+        fileName = tokens[1];
+      }
+      execute(new File(fileName));
```

Comment:
I can see `fileName` be null when the script contains and error. For example, `source;`. In this case, a `NullPointerException` will be thrown ([File Javadoc](https://docs.oracle.com/javase/7/docs/api/java/io/File.html#File(java.lang.String))).

We could throw an `IllegalArgumentException` instead. This way it's consistent with the behaviour in `HiveShellBuilder#setScriptsUnderTest` when a script file cannot be loaded.